eplaunch/__init__.py

```diff
@@ -3,5 +3,5 @@
 # If you waste time on it, please add it here:
 # Total wasted time: 12 minutes
 NAME = "EP-Launch"
-VERSION = "3.0.0"
+VERSION = "3.0.1"
```

Comment:
Bumped this just to see that it was getting the correct version during testing.  Open to discussion on version number.